### DISCUSSION TITLE
Unable to Search for Cities

### DISCUSSION THREAD

**Discussion by Linux-Geezer** [dglent/meteo-qt#116] [Q&A] [answered]
I installed meteo-qt from the Debian Unstable repos. When I attempt to search for my city, I get the following error:

_Error: 401 Unauthorized
Try again later_

I'm wondering if the issue is with the OpenWeather ID. I noticed that there is a field for an OpenWeather ID that is pre-filled with an ID. Do users need to obtain their own ID?

**Comment by dglent** [comment, marked as answer]
Check if you have the latest version first 3.0
If not, you need to update to the latest version or try to create your personal ID

**Reply by Linux-Geezer** [reply]
My apologies; I should have stated the version I'm using. The latest version available from the Debian Unstable repos is v2.1-1. I'll try obtaining my own personal ID. I kind of hate having to do that...yet another online account to manage.

**Reply by Linux-Geezer** [reply]
Obtaining and using my own OpenWeather API ID worked. Temp for my area seems to be a bit off compared to data from the NWS (National Weather Service in the U.S.). App is really nice though.

**Reply by dglent** [reply]
The latest version works with thé default key you don't need to create one. But sometimes someone abuses the usage of  the key some where else and they Block it from openweathermap server.

For the data, you can comparé directly with thé Web site of openweathermap to see if there is any incohérence between thé Web site and meteo-qt, after you can report to the openweathermap support to fix it.

Thanks 😉

**Reply by Linux-Geezer** [reply]
I retract my statement about the temperature accuracy. The NWS (U.S.) data is actually for an airport about 10 miles from where I live. The Meteo data is closer to my outdoor thermometer reading.
